File: Android/app/src/main/java/com/chameleon/tollgate/util/tollgateLog/dto/LogLevel.java
package com.chameleon.tollgate.util.tollgateLog.dto;

import lombok.Getter;

@Getter
public enum LogLevel {
    INFO("Info", 100),
    WARN("Warn", 200),
    ERROR("Error", 300);

    private final String level;
    private final int code;
    private LogLevel(String level, int code){ 
    	this.level = level; 
    	this.code = code;
    }
}
		

File: Client/AuthClient/AuthClient/tollgate/util/tollgateLog/dto/LogLevel.cs
namespace AuthClient.tollgate.util.tollgateLog.dto
{
    class LogLevel
    {
        public static LogLevel INFO
        {
            get
            {
                return new LogLevel("Info", 100);
            }
        }
        public static LogLevel WARN
        {
            get
            {
                return new LogLevel("Warn", 200);
            }
        }
        public static LogLevel ERROR
        {
            get
            {
                return new LogLevel("Error", 300);
            }
        }

        private string level;
        private int code;
        LogLevel(string level, int code)
        {
            this.level = level;
            this.code = code;
        }
        public int getCode()
        {
            return code;
        }
        public string getMessage()
        {
            return level;
        }
    }
}
